deploy/kubernetes/docker/build.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+function exit_with_usage() {
+  set +x
+  echo "./build.sh - Tool for building docker images of Remote Shuffle Service"
+  echo ""
+  echo "Usage:"
+  echo "+------------------------------------------------------------------------------------------------------+"
+  echo "| ./build.sh [--hadoop-version <hadoop version>] [--registry <registry url>]                           |"
+  echo "+------------------------------------------------------------------------------------------------------+"
+  exit 1
+}
+
+UNKNOWN_REGISTRY="UNKNOWN_REGISTRY"
+REGISTRY=$UNKNOWN_REGISTRY
+
+HADOOP_VERSION=3.2.3
```

Comment:
Should we use Hadoop version 2.8.5? Because we use the version in other modules of Uniffle, we should keep consistent with them.